From e374b4f1195256471bb2d6c6cfa113c89cba07ff Mon Sep 17 00:00:00 2001
From: Richard Zhu <hongxing.zhu@nxp.com>
Date: Thu, 2 Nov 2017 13:46:20 +0800
Subject: [PATCH 2725/5242] MLK-16754 ARM64: dts: imx8qm: enable the smmu on
 sata

commit  2aa6a37dbb30226b6115d354da6af91f1d816ddd from
https://source.codeaurora.org/external/imx/linux-imx.git

enable the smmu on sata

BuildInfo:
- SCFW 9559d5ec, IMX-MKIMAGE 06bc2767, ATF
- U-Boot 2017.03-imx_v2017.03_4.9.51_imx8_beta1+g6dc7b0f

Signed-off-by: Richard Zhu <hongxing.zhu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
index cd7ae5d..2189029 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
@@ -2950,6 +2950,7 @@
 				"phy_pclk0", "phy_pclk1";
 		hsio = <&hsio>;
 		power-domains = <&pd_sata0>;
+		iommus = <&smmu 0x13 0x7f80>;
 		status = "disabled";
 	};
 
-- 
1.7.9.5

